README.md

```diff
@@ -7,7 +7,41 @@ This repository provides KubeVirt-specific Tekton tasks, which focus on:
 - Executing commands in VMs
 - Manipulating disk images with virt-customize [WIP]
 
-## Usage and Deployment
+## Deployment
+
+### On Kubernetes
+
+Run the following command to install KubeVirt Tekton tasks in an active namespace.\
+Note: this needs to be run in every namespace the tasks will be used in (due to rbac privileges). 
+
+```bash
+kubectl apply -f https://raw.githubusercontent.com/kubevirt/kubevirt-tekton-tasks/main/manifests/kubernetes/namespace-rbac/kubevirt-tekton-tasks.yaml
+```
+
+If the pipeline needs to access resources (VMs, PVCs, etc.) across different namespaces (cluster privileges), the following commands need to be run instead.
+
+```bash
+TARGET_NAMESPACE="$(kubectl config view --minify --output 'jsonpath={..namespace}')"
+wget -qO - https://raw.githubusercontent.com/kubevirt/kubevirt-tekton-tasks/main/manifests/kubernetes/cluster-rbac/kubevirt-tekton-tasks.yaml | sed "s/TARGET_NAMESPACE/$TARGET_NAMESPACE/" | kubectl apply -f -
+```
+
+### On Openshift
+
+Run the following command to install KubeVirt Tekton tasks in an active namespace. These also include OpenShift specific tasks.\
+Note: this needs to be run in every namespace the tasks will be used in (due to rbac privileges).
+
+```bash
+kubectl apply -f https://raw.githubusercontent.com/kubevirt/kubevirt-tekton-tasks/main/manifests/openshift/namespace-rbac/kubevirt-tekton-tasks.yaml
+```
+
+If the pipeline needs to access resources (VMs, PVCs, etc.) across different namespaces (cluster privileges), the following commands need to be run instead.
+
+```bash
+TARGET_NAMESPACE="$(kubectl config view --minify --output 'jsonpath={..namespace}')"
+wget -qO - https://raw.githubusercontent.com/kubevirt/kubevirt-tekton-tasks/main/manifests/openshift/cluster-rbac/kubevirt-tekton-tasks.yaml | sed "s/TARGET_NAMESPACE/$TARGET_NAMESPACE/" | kubectl apply -f -
```

Comment:
it would be probably the best to link to actual releases instead of the main branch. 

Thoughts on doing actual releases from now on? 
I guess the disadvantage is that we have 4 manifest flavors and 4 ways of deployment.

I could do a first release after this PR with `manifests` folder is merged .

Reply:
I don't understand how having 4 manifests is related to have formal releases.
Your typical user would like to install a tagged release, and your script should probably download the latest tagged manifest.

Reply:
I guess it is not related to having these releases. I just wanted to point out, that it might be confusing for the consumer that each release has 4 different flavors.